README.md

```diff
@@ -25,19 +25,24 @@ app.use(expressLogging(logger));
 app.listen(3000);
 ```
 
-## Extended usage with blacklist
+## Extended usage with options
 
-An optional argument `blacklist` is available to prevent some resources from being logged (for example, static resources). This argument is an array of strings. If the URL path starts with any of the elements of the blacklist array, then the logging of this request/response is ignored.
+An optional argument `options` can customize enhanced aspects for the logging. This argument is an object with the following elements:
 
-The following example would ignore any resource available at either `/images` or `/html`.
+ - `blacklist` is available to prevent some resources from being logged (for example, static resources). This argument is an array of strings. If the URL path starts with any of the elements of the blacklist array, then the logging of this request/response is ignored.
+ - `policy` is a string to customize how the info is logged. It supports two values: `message` or `params`. The former serializes all the log entry into a single string message. The latter passes to the logger an object with the log entry parameters and a second argument with the message; this policy is useful in order to process these parameters by systems like logstash. The default value is `message`.
```

Comment:
It's not very beautiful but I agree that it is the only way to keep backwards compatibility. Do we want backwards compatibility for any reason? Maybe for those using logops 0.x, right?